src/components/shared/SearchInput.js

```diff
@@ -115,7 +119,7 @@ class SearchInput extends Component {
         placeholder,
         value: query,
         spellCheck: false,
-        ref: "input"
+        ref: c => (this.input = c)
```

Comment:
Correction. Seems like you can't add the property to the refs object via a function ref.